pkg/cmd/pipelinerun/logs.go

```diff
@@ -80,10 +80,10 @@ Show the logs of PipelineRun named 'microservice-1' for all Tasks and steps (inc
 	c.Flags().BoolVarP(&opts.Last, "last", "L", false, "show logs for last PipelineRun")
 	c.Flags().BoolVarP(&opts.Fzf, "fzf", "F", false, "use fzf to select a PipelineRun")
 	c.Flags().BoolVarP(&opts.Follow, "follow", "f", false, "stream live logs")
+	c.Flags().BoolVarP(&opts.Timestamps, "timestamps", "", false, "show logs with timestamp")
```

Comment:
should we rename the flag to probably `--show-timestamps` or something of that sort?
cc @piyush-garg @chmouel @pradeepitm12 @vdemeester

Reply:
makes sense. Thanks Chmouel 😄